src/Uno.UI/Helpers/WinUI/DispatchHelper.cs

```diff
@@ -49,16 +49,17 @@ public DispatcherHelper(DependencyObject dependencyObject = null)
 
 		public void RunAsync(Action func, bool fallbackToThisThread = false)
 		{
-			if (dispatcherQueue != null)
+			// TODO: Uno specific - dispatcher queue is not implemented yet
+			if (false)//dispatcherQueue != null)
 			{
-				var result = dispatcherQueue.TryEnqueue(() => func());
-				if (!result)
-				{
-					if (fallbackToThisThread)
-					{
-						func();
-					}
-				}
+				//var result = dispatcherQueue.TryEnqueue(() => func());
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5156819)